src/views/PanelViews/ManagerViews/components/ManagerFormComponent/ManagerFormComponent.js

```diff
@@ -0,0 +1,128 @@
+import React, { useState } from 'react';
+import { employeeStatus } from '../../../shared';
+import { managerFormModes } from './shared';
+
+export function ManagerFormComponent({
+    submitButtonText,
+    formSubmitHandler = () => {},
+    formMode = managerFormModes.WITH_IS_ACTIVE_FIELD,
+    initialValues = {
+        name: '',
+        surname: '',
+        phoneNumber: '',
+        email: '',
+        login: '',
+        password: '',
+        isActive: employeeStatus.ACTIVE,
+    },
+}) {
+    const [name, setName] = useState(initialValues.name);
```

Comment:
To avoid this many states you may create just one state for `formValues` and feed it with initial values as default.